python/ray/scripts/scripts.py

```diff
@@ -800,6 +802,11 @@ def stop(force, verbose, log_style, log_color):
             cli_logger.warning("Try running the command again, or use `{}`.",
                                cf.bold("--force"))
 
+    try:
+        os.remove("/tmp/ray/current_cluster")
```

Comment:
if file exists remove?

Reply:
I think this is the properly pythonic way to do it. https://docs.python.org/3/glossary.html#term-eafp

Reply:
I am not sure this is what it means though, you also have code with if exists and avoids an exceptional case, but won't push too hard.